csv-docs.md

```diff
@@ -0,0 +1,44 @@
+# elbencho CSV Results File
+
+elbencho supports operations against both directories (for example `--mkdirs`) and files (for example `--write`). Each elbencho operation however only deals with one of these so the generic name `entity` is used in the CSV fields. See the `operation` column for the row to determine if the operation was working with directory or file `entities`.
```

Comment:
just a small typo: entry (as in "directory entry") instead of "entity"

Reply:
Oh good catch - I just force-pushed up a fix for this.